models/nft/arbitrum/nft_arbitrum_transfers.sql

```diff
@@ -19,17 +19,25 @@
 , 1 AS amount
 , t.from
 , t.to
+, at.from AS executed_by
 , t.evt_tx_hash AS tx_hash
 , 'arbitrum' || t.evt_tx_hash || '-erc721-' || t.contract_address || '-' || t.tokenId || '-' || t.from || '-' || t.to || '-' || '1' || '-' || t.evt_index AS unique_transfer_id
 FROM {{ source('erc721_arbitrum','evt_transfer') }} t
     {% if is_incremental() %}
         ANTI JOIN {{this}} anti_table
             ON t.evt_tx_hash = anti_table.tx_hash
     {% endif %}
+INNER JOIN arbitrum.transactions at ON at.block_number = t.evt_block_number
```

Comment:
there are still references that need jinja syntax -- appears to be multiple per blockchain

Reply:
Sorry about this, I rushed it thinking there was 1 to be converted by file, I was more careful this time and I'm pretty sure I got all of them.

Reply:
It seems to fail because of the 1h ci timeout but I'm not sure about extending the timeout here because it may need a high runtime because it runs for every single evm chain and L2. What is the best path forward here?

Reply:
we'll give it a shot with increased timeout period, otherwise, i'll have to try and run locally or take the risk to push to prod & monitor there.

Reply:
it ran in less than an hour, weird. i reverted the timeout back and i think we're good to merge!